cScripts/cScripts_pylons.sqf

```diff
@@ -176,5 +176,260 @@
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200]
+            ]]
+        ]]
+    ]],
+    ["rhsusf_M1165A1_GMV_SAG2_Mk19_M240_base", [
+        ["default",[
+            ["displayName", "Default"],
+            ["icon", ""],
+            ["loadout",[
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["RHS_48Rnd_40mm_MK19_M430A1",[0],100],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[2],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200],
+                ["rhs_mag_762x51_M240_200",[3],200]
+            ]]
+        ]]
+    ]],
+    ["B_AFV_Wheeled_01_up_cannon_F", [
+        ["default",[
+            ["displayName", "Heavy HEATFS-T"],
+            ["icon", ""],
+            ["loadout",[
+                ["8Rnd_120mm_HEAT_MP_T_Red",[0],8],
+                ["8Rnd_120mm_HEAT_MP_T_Red",[0],8],
+                ["12Rnd_120mm_APFSDS_shells_Tracer_Red",[0],8],
```

Comment:
12 round we use 8 is it on purpose?

Reply:
Yeah bc they have a specific amount of rounds that they want per mag. They wanted 16 heat and 8 APFSDS